test/k8sT/Policies.go

```diff
@@ -98,6 +98,9 @@ var _ = Describe("K8sPolicyTest", func() {
 		daemonCfg = map[string]string{
 			"global.tls.secretsBackend": "k8s",
 			"global.debug.verbose":      "flow",
+			// enable hubble server and the CLI client
+			"global.hubble.enabled":     "true",
+			"global.hubble.cli.enabled": "true",
```

Comment:
Note: You will have to cleanup the Hubble Client pods spawned by this in `AfterAll`. See https://github.com/cilium/cilium/pull/11687